modules/quickstart-public-cloud/pages/setup.adoc

```diff
@@ -1,54 +1,39 @@
 [[quickstart-publiccloud-setup]]
-= Setting Up
+= Setting up
 
-We have tested these instructions on Amazon EC2, Google Compute Engine, and Microsoft Azure, but they should work on other providers as well, with some variation.
+This guide shows you the fastest way to get {productname} up and running in a public cloud using on-demand or BYOS services.
+We have tested using {productname} on Amazon EC2, Google Compute Engine, and Microsoft Azure, but they should work on other providers as well, with some variation.
 
-You need to start by logging in to your chosen public cloud provider, and launching the {productname} instance.
+There are three main methods of using {productname} on a public cloud service.
 
-Depending on the public cloud you are using, you can usually locate the {productname} Server images by searching for ``suse manager``.
-In EC2, you need to search within the Community AMIs.
-In GCE and Azure, search the marketplace.
-
-As you prepare your new instance, pay attention to these settings:
-
-For hardware, select an instance with at least:
-
-* 32{nbsp}GB RAM
-* 8 cores
-
-For more information about hardware requirements, see xref:installation:hardware-requirements.adoc[].
+Bring your own subscription (BYOS)::
+Most public cloud providers make a BYOS image of {productname} available.
+This means you do not need to install {productname}, just set up the server.
+You will need to have {suse} product entitlements before you begin, and there are some additional setup steps required.
+The public cloud documentation in the {productname} documentation suite assumes you are using this method.
 
-For network security and access settings, you need to enable HTTPS access.
-This allows you to access the {productname} {webui}.
+Virtual machine on public cloud::
+In this method, you subscribe to a public cloud service, and install {productname} in a virtual machine, using the unified installer.
+You will need to have {suse} product entitlements before you begin.
+You can do this by following all the same instructions as you would for any local {productname} installation.
 
-In EC2 and Azure, create a new security group, and add inbound and outbound rules for HTTPS.
-In GCE, check the ``Allow HTTPS traffic`` box under the ``Firewall`` section.
+On-demand {sles} pay as you go (PAYG) image::
+Most public cloud providers make a BYOS image of {sles} available.
```

Comment:
The user would expect information about PAYG first here, imo.

Reply:
I was trying to de-proritise it, because BYOS is the preferred method. There is some discussion that we should not mention PAYG at all, since it involves additional subs.

Reply:
Yes, but what confuses me is that the label (l. 20) announces something about PAYG, but then you start with saying sth. about BYOS (l. 21).

Reply:
Yeah, coz it's BYOS image of *SLES*, not SUMA.